src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/InputMethodEditor.cs

```diff
@@ -2139,5 +2141,75 @@ public static void SetFloatingDragEnd()
                 throw InputMethodExceptionFactory.CreateException(error);
             }
         }
+
+        /// <summary>
+        /// Notifies the changed language of the input panel to the the associated text input UI control.
+        /// </summary>
+        /// <privilege>
+        /// http://tizen.org/privilege/ime
+        /// </privilege>
+        /// <remarks>
+        /// LanguageRequestedCallback is raised after this API is called when the App requests changed language information.
```

Comment:
The above sentence is hard to understand.
Please answer the following questions.
1.  When does an app request to change the language with IME panel or application control, is this API called automatically-It means app control calls this API internally?
2. Can developers call this API directly in their apps?

If the answers of these questions are "YES", you can change the sentence as follow:
```
        /// When an app requests to change the language, this API is called and then LanguageRequestedCallback is raised.
```

Reply:
1. IME can't forward to directly language information to widget when language is changed.
So widget uses LanguageRequiredCallback to get changed language information when the SendLanguageUpdated() was called.
2. Only the IME app can call this API.